modules/nextflow/src/main/groovy/nextflow/container/CharliecloudBuilder.groovy

```diff
@@ -46,6 +47,9 @@ class CharliecloudBuilder extends ContainerBuilder<CharliecloudBuilder> {
         if( params.containsKey('runOptions') )
             addRunOptions(params.runOptions.toString())
 
+        if( params.containsKey('readOnlyInputs') )
+            this.readOnlyInputs = params.readOnlyInputs?.toString() == 'true'
+
```

Comment:
I think you got some unrelated commits in your branch, something might have messed up while merging from upstream

Reply:
I just reverted all these unexpected changes.